uaclient/config.py

```diff
@@ -828,10 +836,33 @@ def simulate_status(
             }
         )
 
+        now = datetime.now(timezone.utc)
         if contract_info.get("effectiveTo"):
             response["expires"] = contract_info.get("effectiveTo")
+            expiration_datetime = util.parse_rfc3339_date(response["expires"])
+            delta = expiration_datetime - now
+            if delta.total_seconds() <= 0:
+                event.warning(
```

Comment:
This will cause the command to fail right ? If yes, I think it should be an error instead

Reply:
Actually it does not fail - as this is just getting the token information, it returns everything even if it is expired - (unlike when attaching, when trying an expired token would error out)

Reply:
As discussed, we will present this as an error. The overall information about a token will still be there, but there will be an error pointing out that the token out of its valid dates.

Reply:
Done